app/views/browse/replica-set.html

```diff
@@ -0,0 +1,151 @@
+<project-header class="top-header"></project-header>
+  <project-page>
+
+    <!-- Middle section -->
+    <div class="middle-section">
+      <div class="middle-container">
+        <div class="middle-header">
+          <div class="container-fluid">
+            <breadcrumbs breadcrumbs="breadcrumbs"></breadcrumbs>
+            <alerts alerts="alerts"></alerts>
+            <div ng-if="!loaded">Loading...</div>
+            <div ng-if="deployment">
+              <h1>
+                {{deployment.metadata.name}}
+                <span
+                  ng-if="deploymentConfigMissing"
+                  class="pficon pficon-warning-triangle-o"
+                  style="cursor: help; vertical-align: middle;"
+                  data-toggle="tooltip"
+                  data-trigger="hover"
+                  title="The deployment's deployment config is missing."
```

Comment:
ah right this needs to work for the old stuff too, nvm leave it till we fix the rest of the Dep->RS stuff